common/src/main/java/org/duracloud/common/constant/Constants.java

```diff
@@ -20,11 +20,11 @@ private Constants() {
         // Ensures no instances are made of this class, as there are only static members.
     }
 
-    public static final String HEADER_PREFIX = "x-dura-meta-";
-
     /**
      * An http header sent by the client that indicates the version of the client.
      */
+    public static final String HEADER_PREFIX = "x-dura-meta-";

```

Comment:
It's not clear why this was moved, but the javadoc refers to the CLIENT_VERSION_HEADER constant below